bin/spregen

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
+
+require 'commander/import'
+require 'spregen'
+
+program :name, 'Spregen'
+program :version, '0.0.1'
+program :description, 'Generate Sprint Reports from your Trello board.'
+
+command :generate do |c|
+  c.syntax = 'spregen generate board'
```

Comment:
"board" here is a parameter from generate command, so, I think [board-name] or just [board] with the brackets will be easier to understand the -help command.
```c.syntax = 'spregen generate [board]'```